test/common/testlib.py

```diff
@@ -717,8 +740,6 @@ def click_system_menu(self, path, enter=True):
         (useful for remote hosts).
         '''
         self.switch_to_top()
-        if self.cdp.mobile:
-            self.click("#nav-system-item")
```

Comment:
This OTOH does have a replacement (although only two commits later, but that's fine).